### PR TITLE
Feature team4 0006

### PR DESCRIPTION
**Pull request by JuanPC06**
LastComponent Desktop

LastDocument 

image

LastDocument - Mobile

image

### CODE REVIEW

File: src/components/LastComponent/lastDocuments.styles.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import { makeStyles } from "@material-ui/styles";
+export const useStyles = makeStyles(() => ({
+  container: {
+    backgroundColor: "#FFFFFF",
```
Comment (Ditmar) [inline src/components/LastComponent/lastDocuments.styles.js:4]: nice pr only one thing use the color from the teme 
`
makeStyles((theme) => ({
backgroundColor: theme.palette.primary.main
})
`

**Review by Ditmar — COMMENTED** [review]
nice work, please check  the comment.

File: src/components/LastComponent/lastDocuments.styles.js (hunk 1 above)
Comment (JuanPC06) [reply src/components/LastComponent/lastDocuments.styles.js]: Correcting theme of my pr Ing review me please

**Review by Ditmar — APPROVED** [review]
nice pr good work

**Pull request merged**